hazelcast-client/src/test/java/com/hazelcast/client/MembershipListenerTest.java

```diff
@@ -62,8 +62,8 @@ public void whenMemberAdded_thenMemberAddedEvent() throws Exception {
         assertTrueEventually(new AssertTask() {
             @Override
             public void run() {
-                assertEquals(1, listener.events.size());
-                MembershipEvent event = listener.events.get(0);
+                assertTrue(1 <= listener.events.size());
```

Comment:
The issue with assertTrue is that if it fails, the message often sucks. Can you improe the message to include the actual events.size? If we get an error, then we know how many there were.